README.md

```diff
@@ -120,18 +120,23 @@ dartdoc:
     - tool-error
 ```
 
+#### dartdoc_options.yaml fields
+
+In general, **paths are relative** to the directory the `dartdoc_options.yaml` the option is defined in
```

Comment:
The grammar here is hard to parse. Maybe

> In general, paths are relative to the directory _of_ the `dartdoc_options.yaml` _file_ _in which_ the option is defined, and should be specified as POSIX paths.

Reply:
done